src/main/java/org/matsim/analysis/RunBerlinAccidents.java

```diff
@@ -42,205 +30,82 @@ public static void main(String[] args) throws IOException {
 		}
 		
 		if ( args.length==0 ) {
-			args = new String[] {"scenarios/berlin-v5.5-1pct/input/berlin-v5.5-1pct.config.xml"}  ;
+			args = new String[] {"https://svn.vsp.tu-berlin.de/repos/public-svn/matsim/scenarios/countries/de/berlin/berlin-v5.5-1pct/input/berlin-v5.5-1pct.config.xml"}  ;
 		}
 		
 	String outputFile = "/../shared-svn/studies/countries/de/accidents/data/output/"; 
 	String landOSMInputShapeFile = "/../shared-svn/studies/countries/de/accidents/data/input/osmBerlin/gis.osm_landuse_a_free_1.shx";  
 	String placesOSMInputFile = "/../shared-svn/studies/countries/de/accidents/data/input/osmBerlinBrandenburg/gis.osm_landuse_a_free_1_GK4.shx";
-	String tunnelLinkCSVInputFile = "D:/GIT/shared-svn/studies/countries/de/accidents/data/input/CSV files/tunnellinks.csv";
+	String tunnelLinkCSVInputFile = "D:/SVN-public/matsim/scenarios/countries/de/berlin/berlin-v5.5-10pct/input/berlin-v5.1.tunnel-linkIDs.csv";

```

Comment:
@Hugo-CM 
This is still referring to a local file location. Should it not be at least '/../../' or better the url, because it is seems to point to the public-svn?

Note: As far as I remember, we at VSP has committed to use '/../../shared-svn/*' instead of '/../' so not each one of us has to adapt the path manually for running it.

Reply:
@kt86
I have corrected it  in my own branch so it follows the convention. thanks for the comment.

Reply:
thanks. Is merged.